components/Footer.tsx

```diff
@@ -1,53 +1,96 @@
-          />
-        </svg>
-      ),
+      name: 'GitHub',
+      href: 'https://github.com/ohcnetwork',
+      icon: 'logos/github-logo.svg',
     },
     {
-      name: "YouTube",
-      href: "https://www.youtube.com/@ohc.network",
-      icon: (props: any) => (
-        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
-          <path
-            fillRule="evenodd"
-            d="M19.812 5.418c.861.23 1.538.907 1.768 1.768C21.998 8.746 22 12 22 12s0 3.255-.418 4.814a2.504 2.504 0 0 1-1.768 1.768c-1.56.419-7.814.419-7.814.419s-6.255 0-7.814-.419a2.505 2.505 0 0 1-1.768-1.768C2 15.255 2 12 2 12s0-3.255.417-4.814a2.507 2.507 0 0 1 1.768-1.768C5.744 5 11.998 5 11.998 5s6.255 0 7.814.418ZM15.194 12 10 15V9l5.194 3Z"
-            clipRule="evenodd"
-          />
-        </svg>
-      ),
+      name: 'Slack',
+      href: 'https://slack.ohc.network/',
+      icon: 'logos/slack-logo.svg',
     },
   ];
+
   return (
-    <footer className="bg-white">
-      <div className="mx-auto mt-32 md:mt-56 max-w-7xl px-6 py-12 md:flex md:items-center md:justify-between lg:px-8">
-        <div className="flex justify-center space-x-6 md:order-2">
-          {footerNavigation.map((item) => (
-            <a
-              key={item.name}
-              href={item.href}
-              className="text-gray-400 hover:text-gray-500"
-            >
-              <span className="sr-only">{item.name}</span>
-              <item.icon className="h-6 w-6" aria-hidden="true" />
-            </a>
-          ))}
+    <footer className="bg-primary-100 mt-20">
+      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-12">
+        <div className="grid grid-cols-1 lg:grid-cols-[1fr,auto,auto] gap-8 lg:gap-16">
+          <div className="max-w-sm">
+            <div className="flex items-center">
+              <Image
+                src="/logos/ohc-logo.png"
+                alt="OHC Network Logo"
+                width={60}
+                height={60}
+                className="h-10 w-auto"
+              />
+              <span className="ml-3 text-primary-900 font-medium text-xl">
+                Open Healthcare Network
+              </span>
+            </div>
+            <p className="mt-4 text-primary-900 text-sm">
+              OHC Network is an open-source platform using AI to revolutionize healthcare.
+            </p>
+          </div>
+
+          <div className="grid grid-cols-2 md:flex md:gap-16 font-medium">
+            <nav className="flex flex-col space-y-3">
+              {navigation.map((item) => (
+                <Link
+                  key={item.name}
+                  href={item.href}
+                  className="text-primary-900 hover:text-primary-600 transition-colors text-sm"
+                >
+                  {item.name}
+                </Link>
+              ))}
+            </nav>
+
+            <div className="flex flex-col space-y-3">
+              {socialLinks.map((item) => (
+                <a
```

Comment:
```suggestion
                  <Link
```
use `Link` even with external links

Reply:
Done